integration-tests/src/test/java/org/web3j/generated/Fibonacci.java

```diff
@@ -33,7 +33,7 @@
  * <p>Generated with web3j version 4.0.1.
```

Comment:
We've been doing this until now, but it really makes any sense to be using a contract wrapper generated for v4.0.1?